core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1761,6 +1776,57 @@ private[spark] class BlockManager(
     blocksToRemove.size
   }
 
+  def decommissionBlockManager(): Unit = {
+    if (!blockManagerDecommissioning) {
```

Comment:
Shall we have a `logDebug` when it's already decommissioning (`blockManagerDecommissioning = true`)?

Reply:
done.

Reply:
done.